### PR TITLE
Update foundation 5 to foundation-sites 6

### PR DESCRIPTION
**Pull request by getglad**
Update __foundation.html
Aligns with new foundation-sites syntax

Update __foundation-index.scss
Add foundation-sites, using foundation-everything mixin

Update __foundation-navbar.html
Aligns with new foundation-sites syntax

Update __navbar.scss
Handful of changes for foundation-sites

Update package.json
Update dependencies to stable releases

### CODE REVIEW

**Comment by getglad** [comment]
The bower dependency needs to also be changed from `foundation` to `foundation-sites`

https://github.com/zurb/foundation-sites

File: package.json [hunk 1]
```diff
@@ -25,29 +25,29 @@
...
     "mkdirp": "~0.5.1",
-    "mocha": "~2.3.3",
+    "mocha": "~2.4.5",
     "mocha-lcov-reporter": "~1.0.0",
-    "mz": "~2.0.0",
-    "recursive-readdir": "~1.2.1",
-    "sinon": "~1.17.1",
+    "mz": "~2.3.1",
+    "recursive-readdir": "~1.3.0",
+    "sinon": "~1.17.3",
```
Comment (zckrs) [inline package.json]: Why this update ?

Comment (getglad) [inline package.json]: Just going off of the depedency monitor. I'm unopininated about it. https://david-dm.org/eleven-labs/generator-gulp-angular#info=devDependencies&view=table

**Comment by Swiip** [comment]
I don't see a version bump in the bower.json, I missed something ?

**Comment by getglad** [comment]
You mean here? https://github.com/Swiip/generator-gulp-angular/blob/master/generators/app/templates/_bower.json#L43

I wasn't 100% sure where that change should go - if that is the correct place, I can add it in.

**Comment by Swiip** [comment]
Yep that's it

**Comment by getglad** [comment]
I have a suggested update to the test-bower package - the first error is failing because it is looking for `/foundation/` and the new directory is `/foundation-sites/`.

I am not sure what the second failure is about, though. I wouldn't have made any changes to `typescript-loader`.
https://travis-ci.org/Swiip/generator-gulp-angular/jobs/108800019#L920
https://github.com/Swiip/generator-gulp-angular/blob/master/test/template/test-package.js#L120

I also reversed the changes to `package.json`, per @zckrs comment.

**Comment by getglad** [comment]
Wanted to just bump this re:update to `test-bower`

**Pull request merged**